src/Utils/Helpers.spec.ts

```diff
@@ -93,41 +94,106 @@ describe('Helpers', () => {
+        expect(result).toMatchObject({
+          windowsVerbatimArguments: false,
+          command: process.execPath,
+          args: expect.arrayContaining([process.argv[1]])
+        });
       });
-    });
 
-    it('should return exec args excluding all app args', () => {
-      const result = Helpers.getExecArgs({ excludeAll: true });
-      expect(result).toMatchObject({
-        command: process.execPath,
-        args: expect.arrayContaining([process.argv[1]])
+      it('should return exec args including extra args', () => {
+        // arrange
+        const extraArgs = ['--run'];
+        // act
+        const result = Helpers.getExecArgs({
+          include: extraArgs,
+          escape: false
+        });
+        // assert
+        expect(result).toMatchObject({
+          windowsVerbatimArguments: false,
+          command: process.execPath,
+          args: [...process.execArgv, ...process.argv.slice(1), ...extraArgs]
+        });
       });
-    });
 
-    it('should return exec args including extra args', () => {
-      const extraArgs = ['--run'];
-      const result = Helpers.getExecArgs({ include: extraArgs });
-      expect(result).toMatchObject({
-        command: process.execPath,
-        args: [...process.execArgv, ...process.argv.slice(1), ...extraArgs]
+      it('should return exec args excluding specific args', () => {
+        // arrange
+        const excessArgs = [...process.argv].slice(-2);
+        // act
+        const result = Helpers.getExecArgs({
+          exclude: excessArgs,
+          escape: false
+        });
+        // assert
+        expect(result).toMatchObject({
+          windowsVerbatimArguments: false,
+          command: process.execPath,
+          args: [
+            ...process.execArgv,
+            ...process.argv.slice(1, process.argv.length - 2)
+          ]
+        });
       });
     });
 
-    it('should return exec args excluding specific args', () => {
-      const excessArgs = [...process.argv].slice(-2);
-      const result = Helpers.getExecArgs({ exclude: excessArgs });
-      expect(result).toMatchObject({
-        command: process.execPath,
-        args: [
-          ...process.execArgv,
-          ...process.argv.slice(1, process.argv.length - 2)
-        ]
+    describe('escaped', () => {
+      // META_CHARS_REGEXP and escapeShellArgument
+      // are borrowed from Helpers class implementation
+      const META_CHARS_REGEXP = /([()\][%!^"`<>&|;, *?])/g;
+
+      const escapeShellArgument = (val: string): string => {
+        val = `${val}`;
+        val = val.replace(/(\\*)"/g, '$1$1\\"');
+        val = val.replace(/(\\*)$/, '$1$1');
+        val = `"${val}"`;
+
+        return val.replace(META_CHARS_REGEXP, '^$1');
+      };
+
+      let spiedProcess!: NodeJS.Process;
+
+      beforeAll(() => {
+        spiedProcess = spy(process);
+        when(spiedProcess.platform).thenReturn('win32');
+      });
+
+      afterAll(() => {
+        reset(spiedProcess);
+      });
```

Comment:
Please remove redundant brackets
```suggestion
      afterAll(() => reset(spiedProcess));
```